.gitignore

```diff
@@ -20,3 +20,6 @@
 .DS_Store
 Thumbs.db
 *.log
+
+# Firebase
+.firebase/*
```

Comment:
> An asterisk "*" matches anything except a slash

I don't think the `*` is needed? And might ignore some folders

`.firebase/` should ignore the whole folder and subdirectories?